factory/22.04/stretch_ros2_humble.repos

```diff
@@ -21,8 +21,8 @@ repositories:
     version: ros2
   stretch_moveit_plugins:
```

Comment:
@hello-atharva stretch_moveit_plugins will likely be housed in stretch_moveit2 as part of the stretch_ros2 repo as we want all moveit related packages to exist together.

Reply:
Should we keep it for the time being till we get that or it's safe to remove it?

Reply:
Let's keep it until we add it to stretch_moveit2.